python/lsst/pipe/supertask/activator.py

```diff
@@ -131,6 +132,34 @@ def activate(self):
         """
         pass
 
+def get_inheritance(stask_class, subclasses=[]):
+    """
+    Get all inheritance classes for a given task to differentiate
+     between SuperTask, CmdLineTask, etc.
+
+    :param stask_class: the output class from pyclbr containing the sub-class information
+    :return: a string with all sub-classes
+    """
+
+    temp_class = stask_class.super[0] # simple inheritance
+    if isinstance(temp_class, pyclbr.Class):
+        subclasses.append(temp_class.name)
+        subclasses = get_inheritance(temp_class, subclasses)
+    else:
+        subclasses.append(temp_class)
+    return subclasses
+
+
+class mock_class(object):
+    """
+    Mock package class to  perform the task discovery for manually entered path to (Super)Task
+    """
+    def __init__(self, path):
+        self.__path__ = [path]
+        self.__name__ = os.path.normpath(path)+os.path.sep 
```

Comment:
Fixed above, the separator is for printing purposes only. I normalize the path, and then add the separator at the end. The reason is that if you enter a path manually, say ~/LSST when doing some printing (or logging, debuging). I will say, I found these modules:
~/LSST.module1
~/LSST.module2
I just wanted it to say, ~/LSST/.module1 instead. 
For a python package it will say:
lsst.pipe.package.module1
the '.' comes from the task discovery which I'm trying to avoid add too much code just to handle manual path packages or python packages.  

I've added extra comments to make this clear